src/Service/OpenPlatform/SearchService.php

```diff
@@ -93,10 +93,6 @@ public function search(string $identifier, string $type, bool $refresh = false):
             throw new OpenPlatformSearchException('Invalid cache argument');
         }
 
-        // We return the material object and not the $item->get() as that
-        // prevents proper testing of the service.
-        $material = null;
-
```

Comment:
These removal sames to be unrelated to the other changes in this PR? Why the removal?

Reply:
From my analysis this was redundant code that was never called.